### ISSUE TITLE
class 'custom_components.eltako.sensor.GatewayReceivedMessagesInActiveSession'> sets an invalid suggested_unit_of_measurement.

### ISSUE THREAD

**Issue by Seqway** [grimmpp/home-assistant-eltako#114]
I followed the instructions of this protocol message in HA:

<class 'custom_components.eltako.sensor.GatewayReceivedMessagesInActiveSession'> sets an invalid suggested_unit_of_measurement. Please create a bug report at https://github.com/grimmpp/home-assistant-eltako/issues. This warning will become an error in Home Assistant Core 2024.5

I run a highe version of HA Core of 2024.5 so maybe this is why I can't connect to my LAN GW.

**Comment by grimmpp** [comment]
I don't think so. Should be actually only a warning. I had this already. I will fix it anyway :-)

**Comment by grimmpp** [comment]
I did a few changes which are right now on main branch. Could you test if it is working now?

**Comment by Blinchik91** [comment]
> I did a few changes which are right now on main branch. Could you test if it is working now?

Ich habe was ähnliche plötzlich bekommen, war vorher nicht so, jetzt aktualisieren sich manche Temperatur Sensoren nicht mehr… nicht alle, das ist auch seltsam, es sind alles die gleichen bei mir.

Weiß nicht was ich machen soll, sie gehen nicht mehr.

ValueError: Entity <class 'custom_components.eltako.sensor.GatewayReceivedMessagesInActiveSession'> suggest an incorrect unit of measurement: Messages.

**Comment by grimmpp** [comment]
Completed in v2.2.1: GatewayReceivedMessagesInActiveSession no longer declares an invalid suggested unit. Covered by tests/test_gateway_entity_registration.py.
